src/pudl/metadata/sources.py

```diff
@@ -21,6 +21,45 @@
         "license_raw": LICENSES["us-govt"],
         "license_pudl": LICENSES["cc-by-4.0"],
     },
+    "eia176": {
+        "title": "EIA Form 176",
+        "path": "https://www.eia.gov/naturalgas/ngqs/all_ng_data.zip",
+        "description": (
+            "The EIA Form 176, also known as the Annual Report of Natural and "
+            "Supplemental Gas Supply and Disposition, describes the origins, suppliers, "
+            "and disposition of natural gas on a yearly and state by state basis."
+        ),
+        "source_file_dict": {
+            "respondents": (
+                "Interstate natural gas pipeline companies; intrastate natural gas "
+                "pipeline companies; natural gas distribution companies; underground "
+                "natural gas storage (UNG) operators; synthetic natural gas plant "
+                "operators; field, well, or processing plant operators that deliver "
+                "natural gas directly to consumers (including their own industrial "
+                "facilities) other than for lease or plant use or processing; "
+                "field, well, or processing plant operators that transport gas to, "
+                "across, or from a State border through field or gathering facilities; "
+                "liquefied natural gas (LNG) storage operators."
+            ),
+            "source_format": "Microsoft Excel (.xls/.xlsx)",
```

Comment:
EIA appears to have fixed the garbled file type issue, and all of the data in the zip file are now actually CSVs.

Reply:
Updated to CSV.